### PR TITLE
fix(password): add @accounts/types to dependencies

### PR DESCRIPTION
**Pull request by kiancross**
Not sure if this should be in `dependencies` or `peerDependencies`, like here:

https://github.com/accounts-js/accounts/blob/0abf18649eddeeb17152caa5e30fdbbfc2c5bf99/packages/graphql-api/package.json#L38

### CODE REVIEW

**Comment by pradel** [comment]
It should be a "peerDependency" like the graphql-api 👍

**Comment by kiancross** [comment]
> It should be a "peerDependency" like the graphql-api

Ok done!

File: packages/password/package.json [hunk 1]
```diff
@@ -29,7 +29,6 @@
   },
   "devDependencies": {
     "@accounts/server": "^0.30.0",
-    "@accounts/types": "^0.30.0",
```
Comment (pradel) [inline packages/password/package.json:32]: can you please rever this line?

**Comment by kiancross** [comment]
Looks like someone else has already submitted a PR that moved that to peer dependencies, as master has been updated.

**Pull request closed without merging**